### ISSUE TITLE
Role "admin" permission issues

### ISSUE THREAD

**Issue by tepene** [hasura/graphql-engine#8417]
When I track a table or a view I get different permission levels for the default role "admin". On some views I only get "select" permission where on others I get "insert, select, update, delete".

Does anybody know how the default permissions are set for the "admin" role? 

### Version Information

Server Version:
v2.2.0 / v2.5.0

### Environment
Hasura CE

### What is the expected behaviour?

![image](https://user-images.githubusercontent.com/54766327/164191778-7c5a8580-e337-475c-8177-85cadeac9e93.png)

### Keywords

role, admin, permission

### What is the current behaviour?

![image](https://user-images.githubusercontent.com/54766327/164191635-b1e5cb95-c592-4338-971c-0a100a3cfb2c.png)

### How to reproduce the issue?

1. Track Table or View
2. Select Table or View
3. Select Permission tab

**Comment by zmitry** [comment]
it depends on a way you defined your view, some views are insertable and others are not.  search for insertable views.

**Comment by tirumaraiselvan** [comment]
Yep, if a view is not "insertable" then you will not get permissions for create/update/delete. This probably needs to be better documented in the console
